### PR TITLE
NAS-113853 / 22.02 / Do not restart all the services when removing vdev from pool

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/disk_/wipe.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def _wipe(self, data):
     @accepts(
         Str('dev'),
         Str('mode', enum=['QUICK', 'FULL', 'FULL_RANDOM'], required=True),
-        Bool('synccache', default=True),
```
Comment (sonicaj) [inline src/middlewared/middlewared/plugins/disk_/wipe.py:57]: We shouldn't change the name of the variable as existing users consuming this via API would be affected by this change

Comment (themylogin) [reply src/middlewared/middlewared/plugins/disk_/wipe.py:57]: These are positional arguments, they are never referenced by their names

File: src/middlewared/middlewared/plugins/s3.py [hunk 2]
```diff
@@ -72,6 +73,7 @@ async def config_extend(self, s3):
         ('edit', {'name': 'secret_key', 'method': lambda x: setattr(
             x, 'validators', [Match(r'^\w+$', explanation='Should only contain alphanumeric characters')]
         )}),
+        ('rm', {'name': 'tls_server_uri'}),
```
Comment (sonicaj) [inline src/middlewared/middlewared/plugins/s3.py:76]: I am not sure i follow this change, we should be able to change this and by doing this - we wouldn't be anymore ?

Comment (themylogin) [reply src/middlewared/middlewared/plugins/s3.py:76]: It was not included in the entry schema so I decided it was not editable. Should it be?

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/s3.py:76]: Yes, it definitely is editable :)

**Review by sonicaj — APPROVED** [review]
Thanks!

**Review by yocalebo — COMMENTED** [review]
Why change from os.SEEK_SET which is literally 0?

**Comment by themylogin** [comment]
@yocalebo this is a position argument, I don't understand why was `os.SEEK_SET` used instead of a zero.

**Comment by themylogin** [comment]
time 0:30

**Pull request merged**